.github/workflows/codecov.yml

```diff
@@ -39,12 +37,17 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
-      - name: Build
-        run: pnpm run build
-
-      - name: Test
-        run: pnpm run test:coverage
```

Comment:
No need to build and test anymore, we just download the artifact that was uploaded during Continuous Integration